### PR TITLE
Improve installation instructions

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

File: docs/source/user_install.md [hunk 1]
```diff
@@ -3,21 +3,19 @@ Installation
...
 
+- With pip:
+
 ``` sourceCode
 pip install ipywidgets
+jupyter nbextension enable --py widgetsnbextension
 ```
 
-or
+When using virtualenv, the <code>--sys-prefix</code> option may be required for
```
Comment (jasongrout) [inline docs/source/user_install.md]: I think sys-prefix could be:
```
`--sys-prefix`
```

File: docs/source/user_install.md [hunk 2]
```diff
@@ -3,21 +3,19 @@ Installation
...
+the extension enabling command in order to keep the environment isolated.
```
Comment (jasongrout) [inline docs/source/user_install.md]: extension-enabling, I think.

**Comment by jasongrout** [comment]
A belated +1 to updating these! I put a few comments inline too.

File: docs/source/user_install.md (hunk 1 above)
Comment (SylvainCorlay) [inline docs/source/user_install.md]: This is unmodified from the original one...

**Pull request merged**